lib-multisrc/wpcomics/src/eu/kanade/tachiyomi/multisrc/wpcomics/WPComics.kt

```diff
@@ -33,6 +38,15 @@ abstract class WPComics(
         .add("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:84.0) Gecko/20100101 Firefox/84.0")
```

Comment:
do the sources work same with mobile useragents? if so probably better to remove hardcoded useragent.